community-membership.md

```diff
@@ -0,0 +1,213 @@
+
+## New contributors
+
+New contributors should be welcomed to the community by existing members,
+helped with PR workflow, and directed to relevant documentation and
+communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the
+principles in this document, familiarity with project organization, roles,
+policies, procedures, conventions, etc., and technical and/or writing ability.
+Role-specific expectations, responsibilities, and requirements are enumerated
+below.
+
+## Member
+
+Members are continuously active contributors in the community. They can have
+issues and PRs assigned to them and pre-submit tests are automatically run for
+their PRs. Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the Operator Framework GitHub organization
+
+### Requirements
+
+- Enabled [two-factor authentication] on their GitHub account.
+- Have made multiple contributions to the project or community. Contribution may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub.
+    - Filing or commenting on issues on GitHub.
+    - Contributing to subproject or community discussions (e.g. meetings, Slack, email discussion
+      forums, Stack Overflow).
+- Subscribed to [operator-framework@googlegroups.com].
+- Actively contributing to 1 or more subprojects.
+- Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
+    - Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating
+      on issues, etc.
+    - Sponsors must be reviewers or approvers in at least 1 OWNERS file either in any repo in the [Operator Framework org],
+      or the org they are sponsoring for.
+      - An approver/reviewer in the [Operator Framework org] may sponsor someone for the [Operator Framework org]
+      as long as it's a project they're involved with.
+- **[Open an issue][membership request] against the operator-framework/org repo**
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist ([preview the current version of the template][membership template])
+   - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship: `+1` or `/lgtm`
+- Once your sponsors have responded, your request will be reviewed by a Operator Framework GitHub org admin. Any missing information will be requested.
+
+### Responsibilities and privileges
+
+- Responsive to issues and PRs assigned to them
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- Members can do `/lgtm` on open PRs.
```

Comment:
IMO just the LGTM's done by reviewers should be counted by the bot.

Reply:
Yeah, assuming we're just trying to mirror the way Kube has stuff set up, shouldn't this be a thing for Reviewers?

Reply:
The goal of letting members `/lgtm` is to allow them to build up to reviewer status by actually getting review experience, therefore I want this requirement to stick around. I don't think tide will merge a PR unless a reviewer assigned to a PR gives a `/lgtm`.

Reply:
I'm not entirely clear on how the current OWNERS are going to be fit into these new roles. Does that mean a person will have to be a Reviewer for their LGTM to actually count for merge purposes, or an Approver? or what?

Reply:
AFAIK your review only counts if you are assigned to a PR.

Reply:
OK. Could we write that down somewhere? I think it's a good idea for the on-ramp for new contributors to be as simple and well-documented as possible. Now that I think about it, a New Contributers's Guide section to the How To Contribute page in the regular docs might be a good place to put this sort of stuff...

Reply:
Wrote down this label requirement. Still need a new contributor's guide though.